superset/assets/src/dashboard/components/PropertiesModal.jsx

```diff
@@ -49,19 +50,25 @@ const defaultProps = {
 class PropertiesModal extends React.PureComponent {
   constructor(props) {
     super(props);
+    this.defaultMetadataValue = JSON.stringify(
```

Comment:
this will throw if the metadata is invalid, is that what we want here? Maybe we should wrap it in a try/catch

Reply:
This is always valid JSON, it's coming from a json object supplied by the backend.